osp/utils/zulip_api.py

```diff
@@ -0,0 +1,37 @@
+import os
+from zulip import Client
+
+from rest_framework.response import Response
+
+client = Client(config_file='~/download')
```

Comment:
add this as var to env and then use it

Reply:
Also document what do we need to do to create the API key and whatever we'll need during deployment

Reply:
It cannot read from a different file. Zulip API file has a different format, I opened and saw! So in comments I have added the directory the file should be kept in.